src/common/options/global.yaml.in

```diff
@@ -3682,6 +3682,29 @@ options:
   min: 896_M
   flags:
   - runtime
+- name: osd_memory_thread_cache_bytes
```

Comment:
One minor quibble here is that we'll need *_thread_cache_bytes settings for each daemon (like the *_memory_target options).  Since most of this is generic, should it just be tcmalloc_thread_cache_bytes and we make sure each daemon uses it if it's compiled with tcmalloc?  We can override global values by putting per-daemon settings in their respective sections right?

I suspect that this is what *_memory_target should have been as well, but I didn't think at the time that it would be used outside of the OSD.

Reply:
Josh @jdurgin proposed that option should be applied to daemons, but not to clients. This can be done, but it will mean that we no longer can apply different defaults for different daemons.